src/public/js/index.js

```diff
@@ -0,0 +1,28 @@
+import getTimeRemaining from './countdown';
+
+const initializeClock = (minutes) => {
+  document.getElementById('start').style.display = 'none';
+  document.getElementById('stop').style.display = 'block';
+  const endtime = Date.parse(new Date()) + (minutes * 60 * 1000); // number of miliseconds until countdown finishes
+  const clock = document.getElementById('clockdiv');
+  const timeinterval = setInterval(function () {
+    const t = getTimeRemaining(endtime);
+    const seconds = t.seconds.toString().length === 1 ? '0' + t.seconds : t.seconds;
+    const minutes = t.minutes.toString().length === 1 ? '0' + t.minutes : t.minutes;
```

Comment:
Would it not be better to write `t.seconds < 10` rather than `t.seconds.toString().length === 1` and the same for mins